Documentation/Wired.md

```diff
@@ -0,0 +1,228 @@
+
+# Wired Assembly
+
+*How to build a cheap **Hotdog**.*
+
+<br>
+
+If you have access to a 3D printer and use a cheap <br>
+Nano clone with `CH340`, the whole thing will run <br>
+you around `$20`, not counting the sleeve itself.
+
+<br>
+
+## Parts
+
+-   Transparent Sleeve
+
+    *I use **[This][Sleeve]** one.*
+
```

Comment:
I prefer no blank lines between list items either (and keeping list items to a single line when possible), mostly for readability when viewed as plain text.

Reply:
How is one stuffed line more readable
then an aligned paragraph though?

Reply:
It's not about the word count, but the fact 
that it's two separate things in one line.

### Guides

To begin with, I would generally reorder
the assembly guides into multiple parts
and not one long one.

Including creating a dedicated 
section for 'things I used'.

Reply:
Not important enough to further argue about. Accepted.